web-bundle/src/main/java/com/graphhopper/replica/GtfsLinkMapper.java

```diff
@@ -116,7 +116,7 @@ public void setGtfsLinkMappings() {
                         stop.stop_lat, stop.stop_lon,
                         nextStop.stop_lat, nextStop.stop_lon
                 );
-                odRequest.setProfile("car_local");
+                odRequest.setProfile("gtfs_link_mapper");
```

Comment:
nit: can we pull this profile name into a constant and use the constant here and where we do the validation?

Reply:
ah sorry for all that trouble, but yeah i like the solution of putting it in `replica-common`! getting those shared usages into a constant is much more important to me than cleaning up the dep directions. using IntelliJ to see all usages of a variable has been super helpful for me during refactors/debugging/understanding how the different parts fit together, especially since the Github web UI doesn't allow you to search the repo since it's a fork

Reply:
Sweet! Totally agree that the benefits outweigh the annoyances of getting this moved into a shared constant, the ease of browsing the code + the safety definitely make sense for something that ends up having this level of importance for commands we rely on often